### ISSUE TITLE
Adjust styles of the "Show password" icon

### ISSUE THREAD

**Issue by outdoor2kode** [WordPress/wporg-two-factor#83]
Ref: https://github.com/WordPress/wporg-two-factor/issues/5#issuecomment-1415629776

- [x] The "Show password" icon is better to be inside the input field.
![image](https://user-images.githubusercontent.com/18050944/225783599-e11de281-1045-46b7-b526-7e1cbf9fadaf.png)
- [x] Have a toggled state for this icon.
![image](https://user-images.githubusercontent.com/18050944/225784464-88346654-25c6-43ef-beb3-411043fe9d68.png)
- [x] The focus style here is a bit awkward. (It could be square if the icon sat inside the input)
![image](https://user-images.githubusercontent.com/18050944/225784440-9a17031c-a5bb-46df-86b8-a99ff5b18e6c.png)
- [x] The input should not be pushed when toggling state.
![Screen Capture on 2023-03-17 at 09-00-48](https://user-images.githubusercontent.com/18050944/225785695-1a9fad0a-5d16-45f2-9225-1583fdf7f27e.gif)

**Comment by outdoor2kode** [comment]
@jasmussen Regarding the style of the unseen toggle, could you provide this one? Thanks.
![image](https://user-images.githubusercontent.com/18050944/225786032-988dc7cb-18eb-4fff-8a11-88a0134ea2f7.png)

It kind of aligns with the style that will be released in 6.3. ([ref](https://core.trac.wordpress.org/ticket/9883#comment:44))
![image](https://user-images.githubusercontent.com/18050944/225786164-12ad9477-868b-4c9a-b5fd-3c9baf293b19.png)

**Comment by jasmussen** [comment]
Actually I think we should go with the closed eye. 

<img width="743" alt="Screenshot 2023-03-17 at 08 22 38" src="https://user-images.githubusercontent.com/1204802/225839351-0c0acf67-12a2-4a79-825e-808196c97057.png">

Here's an SVG:

```
<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" fill="none"><path fill="#1E1E1E" d="M4.67 10.664s-2.09 1.11-2.917 1.582l.494.87 1.608-.914.002.002c.343.502.86 1.17 1.563 1.84.348.33.742.663 1.185.976L5.57 16.744l.858.515 1.02-1.701a9.1 9.1 0 0 0 4.051 1.18V19h1v-2.263a9.1 9.1 0 0 0 4.05-1.18l1.021 1.7.858-.514-1.034-1.723c.442-.313.837-.646 1.184-.977.703-.669 1.22-1.337 1.563-1.839l.002-.003 1.61.914.493-.87c-1.75-.994-2.918-1.58-2.918-1.58l-.003.005a8.29 8.29 0 0 1-.422.689 10.097 10.097 0 0 1-1.36 1.598c-1.218 1.16-3.042 2.293-5.544 2.293-2.503 0-4.327-1.132-5.546-2.293a10.099 10.099 0 0 1-1.359-1.599 8.267 8.267 0 0 1-.422-.689l-.003-.005Z"/></svg>
```
